.github/workflows/integration.yml

```diff
@@ -86,7 +90,14 @@ on:
           - letsencrypt_staging
           - letsencrypt_production
         default: letsencrypt_production
-
+      is_pre_installed:
```

Comment:
I would add this as an `install_profile`, something like `minimal-pre-installed`. And maybe set it as the default?

Reply:
would this in addition to a workflow parameter or instead of the workflow parameter?